OpenRA.Mods.Common/Traits/AppearsOnRadar.cs

```diff
@@ -40,7 +40,22 @@ public void Created(Actor self)
 
 		public IEnumerable<Pair<CPos, Color>> RadarSignatureCells(Actor self)
 		{
-			var color = modifier != null ? modifier.RadarColorOverride(self) : self.Owner.Color.RGB;
+			var player = self.World.RenderPlayer ?? self.World.LocalPlayer;
+			var color = self.Owner.Color.RGB;
+
+			if (Game.Settings.Game.AllianceDisplay)
+				if (self.Owner.PlayerName != "Neutral")
```

Comment:
Not a good idea to compare player names.  Check out the health bars or other parts of the UI to see how to compare against stances instead.